owncloudData/src/test/java/com/owncloud/android/data/storage/ScopedStorageProviderTest.kt

```diff
@@ -0,0 +1,330 @@
+package com.owncloud.android.data.storage
+
+import android.content.Context
+import android.net.Uri
+import com.owncloud.android.domain.transfers.model.OCTransfer
+import com.owncloud.android.testutil.OC_FILE
+import com.owncloud.android.testutil.OC_SPACE_PROJECT_WITH_IMAGE
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.verify
+import junit.framework.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import java.io.File
```

Comment:
I think you added some other imports after reordering them, now they should be reordered again 😁

Reply:
Done